demos/browser/app/meetingReadinessChecker/meetingReadinessChecker.ts

```diff
@@ -91,24 +96,33 @@ export class DemoMeetingApp {
     this.defaultBrowserBehaviour = new DefaultBrowserBehavior();
     this.meeting = `READINESS_CHECKER-${uuidv4()}`;
```

Comment:
this.meeting and name should move to startMeetingAndInitializeMeetingReadinessChecker. We dont need it in the first page.